### ISSUE TITLE
Error log flooded with ActionView::MissingTemplate

### ISSUE THREAD

**Issue by hut8** [tenex/rails-assets#108]
Someone is hitting `/` with `Accept-Content-Type: application/json`, and it's flooding our error logs with:

```
Missing template main/home, application/home with {:locale=>[:en], :formats=>[:json], :variants=>[], :handlers=>[:erb, :builder, :raw, :ruby, :slim, :coffee]}. Searched in: * "/home/rails-assets/rails-apps/rails-assets/releases/20160513135658/app/views" * "/home/rails-assets/rails-apps/rails-assets/shared/bundle/ruby/2.2.0/gems/honey_badger-0.0.4/app/views"
```

We should be throwing `UnknownFormat` here anyway, and also handling it somehow, returning a 406 instead of 500.

```
  rescue_from ActionController::UnknownFormat, with: :handle_it_somehow
```

**Comment by hut8** [comment]
https://github.com/tenex/rails-assets/tree/lb/improve-unknown-format-error-handling

**Comment by hut8** [comment]
Resolved via #109
